src/fiatconnect/saga.test.ts

```diff
@@ -0,0 +1,92 @@
+import { expectSaga } from 'redux-saga-test-plan'
+import { select } from 'redux-saga/effects'
+import {
+  fiatConnectCashInEnabledSelector,
+  fiatConnectCashOutEnabledSelector,
+} from 'src/app/selectors'
+import { fetchQuotes } from 'src/fiatconnect'
+import { handleFetchFiatConnectQuotes } from 'src/fiatconnect/saga'
+import {
+  fetchFiatConnectQuotes,
+  fetchFiatConnectQuotesCompleted,
+  fetchFiatConnectQuotesFailed,
+} from 'src/fiatconnect/slice'
+import { CICOFlow } from 'src/fiatExchanges/utils'
+import { getLocalCurrencyCode } from 'src/localCurrency/selectors'
+import { userLocationDataSelector } from 'src/networkInfo/selectors'
+import { CiCoCurrency } from 'src/utils/currencies'
+import { currentAccountSelector } from 'src/web3/selectors'
+import { mockFiatConnectQuotes } from 'test/values'
+
+jest.mock('src/fiatconnect')
+
+describe('Fiatconnect saga saga', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+  describe('Handles fetching quotes', () => {
+    it('saves the dapps and categories', async () => {
+      ;(fetchQuotes as jest.Mock).mockImplementation(() => mockFiatConnectQuotes)
```

Comment:
you can use the [`mocked` helper](https://kulshekhar.github.io/ts-jest/docs/26.5/guides/test-helpers#mockedtitem-t-deep--false) instead of `as`. Looks like this is available on jest directly since 27.4, but we can use this helper from ts-jest